BoxContentSDK/BoxContentSDK/Operations/BOXAPIOperation.m

```diff
@@ -53,7 +53,7 @@ static BOOL BoxOperationStateTransitionIsValid(BOXAPIOperationState fromState, B
 
 @interface BOXAPIOperation()
 
-- (void)cancelConnection;
+- (void)cancelSessionTask;
```

Comment:
yeah, it is cancelling session task explicitly and potentially producing resume data